Core/include/Acts/Geometry/DetectorVolume.hpp

```diff
@@ -310,6 +315,8 @@ class DetectorVolume : public std::enable_shared_from_this<DetectorVolume> {
   /// Const access to the detector
   const Detector* detector() const;
 
+  const std::shared_ptr<BoundingBox> getBoundingBox() const;
```

Comment:
Do you need to expose this as a `shared_ptr`? In this form, the `const` on the left also doesn't do anything.
Would the following work?
```suggestion
  const BoundingBox& getBoundingBox() const;
```

Reply:
Done